### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by gemguardian**
Hello Nicolas,
Enclosed you find the Dutch translation as discussed during the Global Moot. 
It took a little bit longer to pull this project off, but I hope you can add the translation to your branch. 

If I missed something (or did something wrong) please let me know. 

Gemma

### CODE REVIEW

File: content/nl/about/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Over
+description: Wat is het "Moodle Gereedschapsgids"?
+---
+
+_Moodle Gereedschapsgids_ is een referentiekaart voor het gebruik van [Moodle][moodle] versie 3 en later, ontworpen voor leerkrachten, opleiders en trainers. Het geeft informatie over de belangrijkste Moodle-tools en welke tool gebruikt moet worden in overeenstemming met het pedagogische doel dat bereikt moet worden. Het vermeldt ook de taxonomische niveaus van Bloom die bereikt kunnen worden bij het gebruik van de verschillende tools.
+
+Een idee van [Joyce Seitzinger](https://twitter.com/catspyjamasnz) van [Eastern Institute of Technology](https://www.eit.ac.nz/) (NZ). In mei 2020 aangepast voor Moodle 3.x door [Nicolas Martignoni][nm], gebaseerd op het werk van versi 2 door [Gavin Henrick](https://twitter.com/ghenrick).
```
Comment (martignoni) [inline content/nl/about/index.md:8]:
```suggestion
Een idee van [Joyce Seitzinger](https://twitter.com/catspyjamasnz) van [Eastern Institute of Technology](https://www.eit.ac.nz/) (NZ). In mei 2020 aangepast voor Moodle 3.x door [Nicolas Martignoni][nm], INSERT HERE CREDITS FOR DUTCH TRANSLATION.
```
See e.g. [Spanish version](https://github.com/martignoni/moodle-tool-guide/blob/master/content/es/about/index.md), line 8.

**Comment by martignoni** [comment]
Hi Gemma,
Thanks for the PR, it seems to me it's OK, and I'm keen to merge it into the main repo for publishing.

Two demands yet:
- Please change the About page (see suggestion inline) with credits for Dutch translation (yourself and others if applicable)
- Could you also translate the following strings needed in the configuration file:
    - "Provided by Nicolas Martignoni – The website moodletoolguide.net is licensed under a Creative Commons BY-NC-SA 4.0 International License."
    - "An idea of Joyce Seitzinger of Eastern Institute of Technology (NZ), adapted by Nicolas Martignoni and translated info Dutch by <INSERT CREDITS HERE (liek above)>."
    - All the keywords (for SEO): Moodle Tools, Moodle Tools Guide, Moodle Tool Guide, Moodle Tool Guide for teachers, Moodle Tools for teachers, Moodle Guide, Moodle Guide for teachers, Moodle Guide for instructors, Moodle Guide for trainers

TIA
Nicolas

File: content/nl/about/index.md (hunk 1 above)
Comment (gemguardian) [reply content/nl/about/index.md]: Hi Nicolas,
I am a little bit new in this all but if I am correct I changed the file as requested. Please let me know if not.
Gemma

**Review by gemguardian — COMMENTED** [review]
Hi Nicolas,
I made the change for the about page. And a seperated pull for the config file. 

Hope that this is okay.

I don't work that much with Github, so if something is not okay please let me know :)

Gem

**Comment by martignoni** [comment]
That's perfect. A big thank you for the 10th MoodleToolGuide language.

**Pull request merged**